eng/runHelixAndNonHelixInParallel.ps1

```diff
@@ -0,0 +1,40 @@
+    [CmdletBinding(PositionalBinding = $false)]
+    Param(
+        [string] $configuration,
+        [string] $buildSourcesDirectory,
+        [string] $customHelixTargetQueue,
+        [switch] $test
+    )
+
+    if (-not$test)
```

Comment:
Should there be a space here?

```suggestion
    if (-not $test)
```

Reply:
My automatic formatter did that. I guess it is a convention somewhere